crates/test-utils/src/fixture.rs

```diff
@@ -107,6 +107,11 @@ pub struct Fixture {
     ///
     /// Syntax: `env:PATH=/bin,RUST_LOG=debug`
     pub env: FxHashMap<String, String>,
+    /// Specifies extra crate-level attributes injected at the top of the crate root file.
+    /// This must be used with `crate` meta.
+    ///
+    /// Syntax: `crate-attrs:no_std,features(f16,f128),crate_type="bin",cfg(target_arch="x86")`
+    pub crate_attrs: Vec<String>,
```

Comment:
Weak preference (I'll let you decide) but I think I prefer non-comma-separated attributes, instead repeat the meta command to insert more attributes.